### PR TITLE
Update OIDC docs

### PR DESCRIPTION
**Pull request by michaellzc**

### CODE REVIEW

File: content/departments/cloud/technical-docs/oidc_site_admin.md [hunk 1]
```diff
@@ -1,107 +1,7 @@
...
+Sourcegraph employees access to managed instances application interface (Web UI) is restricted to essential personnel only. This ensures Sourcegraph is able to help customers troubleshoot issues and deliver a smooth experience. We utilize OpenID Connect to enable Sourcegraph employees access to customer instance to make sure there is an audit trail for every access.
 
-[RFC 576: Allow disabling sign-up for OpenID Connect auth providers](https://docs.google.com/document/d/1Op6jum_SQJSU5KeJlJJEED1QYFdXOO30botb3BgEO6U/edit) provided the necessary changes to allow Sourcegraph to use their internal Okta Auth provider as a way to grant Site Admin access to managed instances. Okta provides 2FA enforcement and audit logs which help improve the security posture of Managed Instance access.
+# Implementation
 
-# Procedure
-
-These instructions are adapted from [Sourcegraph Docs Site - User authentication (SSO) instructions](https://docs.sourcegraph.com/admin/auth#openid-connect).
-
-## Adjusting License for increase seat count
```
Comment (RafLeszczynski) [inline content/departments/cloud/technical-docs/oidc_site_admin.md:11]: Is the adjusting of licenses no longer needed?

File: content/departments/cloud/technical-docs/oidc_site_admin.md [hunk 2]
```diff
@@ -1,107 +1,7 @@
...
-
-> **NOTE:** The convention for Sourcegraph Management usernames is `sourcegraph-management-<first>-<last>`.
-
-Add Sourcegraph Teammates as required. Use the convention for usernames and use their corresponding @sourcegraph.com email address.
-
-- Customer Support - Application Engineers
-  - A subset of Application Engineers have access to managed instances. [They are listed in the Support Process page for Managed Instances](../../ce-support/support/process/support-managed-instances.md#accessing-managed-instances)
-- Delivery Team
-  - [All software engineers for on-call rotation](../index.md#team)
+[RFC 576: Allow disabling sign-up for OpenID Connect auth providers](https://docs.google.com/document/d/1Op6jum_SQJSU5KeJlJJEED1QYFdXOO30botb3BgEO6U/edit) provided the necessary changes to allow Sourcegraph to use their Google Workspace account (federated by internal Okta Auth provider) as a way to grant Site Admin access to managed instances. Okta provides 2FA enforcement and audit logs which help improve the security posture of Managed Instance access which is significantly more secure comparing to the default basic auth provider.
```
Comment (RafLeszczynski) [inline content/departments/cloud/technical-docs/oidc_site_admin.md:7]: Is the link to RFC enough or should we also have some sort of playbook in the handbook like the one that is just removed?

Comment (michaellzc) [reply content/departments/cloud/technical-docs/oidc_site_admin.md:7]: this is not necessary, we already automate everything that is described here and it's run once during instance initilization.

File: content/departments/cloud/technical-docs/oidc_site_admin.md (hunk 1 above)
Comment (michaellzc) [reply content/departments/cloud/technical-docs/oidc_site_admin.md:11]: I just moved it to a FAQ below

**Pull request merged**